executor.py

```diff
@@ -147,4 +140,22 @@ def fill_embedding(self, docs: Optional[DocumentArray], **kwargs):
             return
 
         for doc in docs:
-            doc.embedding = self._storage[doc.id].embedding
+            doc.embedding = self._index[doc.id].embedding
+
+    @requests(on='/dump')
+    def dump(self, docs: Optional[DocumentArray]=None, **kwargs):
+        """dump indexed Documents to disk
+        """
+        if docs:
+            self._index.extend(docs)
+        bytes = self._index.to_bytes(protocol=self.protocol, compress=self.compress)
+        with open(os.path.join(self.workspace, SimpleIndexer.FILE_NAME), 'wb') as f:
+            f.write(bytes)
+
+    @requests(on='/load')
+    def load_from_disk(self, docs: Optional[DocumentArray]=None, **kwargs):
+        with open(os.path.join(self.workspace, SimpleIndexer.FILE_NAME), 'rb') as f:
+            self._index = DocumentArray.from_bytes(f, protocol=self.protocol, compress=self.compress)
+        if docs:
+            self._index.extend(docs)
```

Comment:
The same as above. I don't see the rationale of having `docs` as an input argument.